tests/unittests/core/types/pointtest.cpp

```diff
@@ -156,6 +156,86 @@ INSTANTIATE_TEST_SUITE_P(PointRotateTest, PointRotateTest, ::testing::Values(
 ));
 // clang-format on
 
+/*******************************************************************************
+ *  Tests for getLength()
+ ******************************************************************************/
+
+struct PointLengthPrecisionTestData {
+  Point input;
+  UnsignedLength length;
+};
+
+class PointLengthPrecisionTest : public ::testing::TestWithParam<PointLengthPrecisionTestData> {};
+
+TEST_P(PointLengthPrecisionTest, testPrecision) {
+  const PointLengthPrecisionTestData& data = GetParam();
+
+  Point p(data.input);
+  EXPECT_NO_THROW(p.getLength());
+  EXPECT_EQ(data.length, p.getLength()) << "Computed: " << p.getLength()->toNm() << " Expected: " << data.length->toNm();
+}
+
+// sqrt2 according wikipedia 1.41421356237309504880168872420969807856967187537694807317667973799
+
+// clang-format off
+INSTANTIATE_TEST_SUITE_P(PointLengthPrecisionTest, PointLengthPrecisionTest, ::testing::Values(
+    //                  {input,                     length}
+    PointLengthPrecisionTestData({Point(0, 0),               UnsignedLength(00)}),
```

Comment:
Ah btw, this octal literal also looks a bit "unusual" :see_no_evil: 

```suggestion
    PointLengthPrecisionTestData({Point(0, 0),               UnsignedLength(0)}),
```

Reply:
Good hit! Thanks !